src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -0,0 +1,3389 @@
+          onAttack {
+
+          }
+        }
+        move "Kindle", {
+          text "120 damage. Discard an Energy from this Pokémon. Then, discard an Energy from your opponent’s Active Pokémon."
+          energyCost R, R, C
+          attackRequirement {}
+          onAttack {
+            damage 120
+          }
+        }
+      };
+      case HOUNDOOM_V_22:
+      return basic (this, hp:HP210, type:R, retreatCost:1) {
+        weakness W
+        move "Searing Flame", {
+          text "20 damage. Your opponent’s Active Pokémon is now Burned."
+          energyCost R
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+        move "Compensation Flame", {
+          text "100+ damage. If any of your [R] Pokémon have damage counters on them, this attack does 100 more damage."
+          energyCost R, R, C
+          attackRequirement {}
+          onAttack {
+            damage 100
+          }
+        }
+      };
+      case TORCHIC_23:
+      return basic (this, hp:HP060, type:R, retreatCost:null) {
+        weakness W
+        move "Call for Family", {
+          text "Search your deck for a Basic Pokémon and put it on your Bench. Then, shuffle your deck."
+          energyCost C
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
+        move "Live Coal", {
+          text "10 damage. "
+          energyCost R
+          attackRequirement {}
+          onAttack {
+            damage 10
+          }
+        }
+      };
+      case COMBUSKEN_24:
+      return evolution (this, from:"Torchic", hp:HP090, type:R, retreatCost:null) {
+        weakness W
+        move "Smash Kick", {
+          text "20 damage. "
+          energyCost R
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+        move "Heat Beak", {
+          text "40 damage. Your opponent’s Active Pokémon is now Burned."
+          energyCost R, C, C
+          attackRequirement {}
+          onAttack {
+            damage 40
+          }
+        }
+      };
+      case BLAZIKEN_25:
+      return evolution (this, from:"Combusken", hp:HP170, type:R, retreatCost:2) {
+        weakness W
+        bwAbility "Double Type", {
+          text "This Pokémon is both [R] and [F] type."
+          actionA {
+          }
+        }
+        move "Turbo Drive", {
+          text "130 damage. Attach a basic Energy card from your discard pile to 1 of your Benched Pokémon."
+          energyCost R, C, C
+          attackRequirement {}
+          onAttack {
+            damage 130
+          }
+        }
+      };
+      case HEATRAN_26:
+      return basic (this, hp:HP130, type:R, retreatCost:4) {
+        weakness W
+        move "Fire Fang", {
+          text "30 damage. Your opponent’s Active Pokémon is now Burned."
+          energyCost R, C
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+      };
```

Comment:
If the following card is the correct one, missing the second attack.

[Heatran](https://www.pokebeach.com/news/0420/038073_P_HIDORAN-2.jpg)

Reply:
Good catch!